tcks/profiles/platform/signaturevalidation/src/main/resources/com/sun/ts/tests/signaturetest/signature-repository/sig-test.map

```diff
@@ -0,0 +1,84 @@
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception, which is available at
+# https://www.gnu.org/software/classpath/license.html.
+#
+# SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+#
+
+###############################################################
+#
+# The signature test mapping file for the JakartaEE TCK.  This file
+# should be formatted as a standard java properties file.  The
+# name is the package name and the value is the version of the
+# package that should be tested by the signature tests.
+#
+# The resultant signature file name has a name that complies
+# with the following format:
+# <package>.sig
+# Given this format, the following example breaks down as follows:
+#    jakarta.servlet=4.0
+#      <package> = jakarta.servlet
+#      <technology_version> = 4.1  (for Servlet)
+#
+#
+# For updates to this file when the JavaEE spec gets updated,
+# as a good rule of thumb, we pull out any packages that end
+# up as being included with the Java SE. So if any of the packages
+# listed below ends up being incorporated into Java SE in the future,
+# that will be a hint that those packages can likely be removed
+# from this sig-test.map file.
+#
+# Packages and versions pulled from JavaEE 11 platform spec.
+#
+# Note: Recording the signatures of a package includes all
+#       child packages.  The signature test tool looks for
+#       the best signature file to use when playing back
+#       signatures.  Meaning if we have a jakarta.servlet
+#       signature file and a jakarta.servlet.jsp signature file,
+#       the signature test tool will use the jakarta.servlet.jsp
+#       signature file to verify the jakarta.servlet.jsp package
+#       signatures even though the jakarta.servlet signature
+#       file contains the jakarta.servlet.jsp package signatures.
+#       The signatures are in both files (since the API Check
+#       tool records child package signatures and there does
+#       not seem to be a way to turn this feature off) but the
+#       jakarta.servlet.jsp signature file can vary independent
+#       of the jakarta.servlet signature file.
+#
+jakarta.annotation=3.0
+jakarta.batch=2.1
+jakarta.data=1.0
+# Part of CDI, but required to be separate for now
+jakarta.decorator=4.1
+jakarta.ejb=4.0
+jakarta.el=6.0
+jakarta.enterprise.concurrent=3.1
+jakarta.enterprise=4.1
+jakarta.faces=4.1
+jakarta.inject=2.0
+jakarta.interceptor=2.2
+jakarta.jms=3.1
+jakarta.json.bind=3.0
+jakarta.json=2.1
+jakarta.mail=2.1
+jakarta.persistence=3.2
+jakarta.resource=2.1
+jakarta.security.auth.message=3.1
+jakarta.security.enterprise=4.0
+jakarta.security.jacc=3.0
+jakarta.servlet.jsp.jstl=4.0
```

Comment:
I think jstl is 3.0 https://jakarta.ee/specifications/tags/

Reply:
Oops, yes good catch.